content/applications/general/users.rst

```diff
@@ -4,143 +4,235 @@
-rules can be applied. Users and access rights can be added and changed at any point.
+Odoo defines a *user* as someone who has access to a database to perform daily tasks. An
+administrator can add as many users as the company needs and, in order to restrict the type of
+information each user can access, rules can be applied. Users and access rights can be added and
+changed at any point.
 
 .. seealso::
    - :doc:`users/language`
    - :doc:`users/access_rights`
+   - :ref:`access-rights/superuser`
+   - :ref:`access-rights/groups`
 
 .. _users/add-individual:
 
 Add individual users
 ====================
 
-Go to :menuselection:`Settings --> Manage Users` and click on *Create*.
+Go to :menuselection:`Settings --> Users --> Manage Users` and click on :guilabel:`Create`.
 
 .. image:: users/manage-users.png
-   :height: 280
-   :alt: View of the settings page emphasizing the manage users field in Odoo
+   :align: center
+   :alt: View of the settings page emphasizing the manage users field in Odoo.
 
-| Fill in the form with the needed information. Under the tab
-  :doc:`Access Rights <users/access_rights>` choose the group within each application the user can
-  have access to.
-| The list of applications shown is based on the applications installed on the database.
+Fill in the form with the needed information. Under the tab :doc:`Access Rights
+<users/access_rights>` choose the group within each application the user can have access to.
 
-.. image:: users/new_user.png
-   :alt: View of a user’s form emphasizing the access rights tab in Odoo
+The list of applications shown is based on the applications installed on the database.
 
-When you are done editing the page and have *Saved* it, an invitation email is automatically sent to
-the user. The user must click on it to accept the invitation and create a login.
+.. image:: users/new-user.png
+   :align: center
+   :alt: View of a user's form emphasizing the access rights tab in Odoo.
+
+After filling out all the necessary fields on the page, click :guilabel:`Save`. Then, an invitation
+email is automatically sent to the user, using the email in the :guilabel:`Email Address` field. The
+user must click on the link included in the email to accept the invitation and to create a database
+login.
 
 .. image:: users/invitation-email.png
-   :alt: View of a user’s form with a notification that the invitation email has been sent in Odoo
+   :align: center
+   :alt: View of a user's form with a notification that the invitation email has been sent in Odoo.
 
-.. note::
-   Remember that subscription prices follow the number of users. Refer to our
-   `pricing page <https://www.odoo.com/pricing>`_
-   for more information.
+.. warning::
+   If the company is on a monthly subscription plan, the database will automatically update to
+   reflect the added user(s). If the company is on a yearly or multi-year plan, an expiration banner
```

Comment:
```suggestion
   reflect the added users. If the company is on a yearly or multi-year plan, an expiration banner
```
I believe we were told not to use (s) and just use the -s. Feel free to double check me on that though